src/main/java/org/reactome/server/controller/DoiController.java

```diff
@@ -0,0 +1,53 @@
+package org.reactome.server.controller;
+
```

Comment:
minor minor minor: just clean up the excessive new lines added along the code...